dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -29,23 +32,16 @@ def save
   # GET api/v1/ml_models/names
   # Retrieve the names, ids and metadata of a user's trained ML models.
   def names
-    user_ml_model_data = UserMlModel.where(user_id: current_user&.id).map {|user_ml_model| {id: user_ml_model.model_id, name: user_ml_model.name, metadata: JSON.parse(user_ml_model.metadata)}}
+    user_ml_model_data = UserMlModel.where(user_id: current_user&.id).
+      map {|user_ml_model| {id: user_ml_model.model_id, name: user_ml_model.name, metadata: JSON.parse(user_ml_model.metadata)}}
     render json: user_ml_model_data.to_json
   end
 
-  # GET api/v1/ml_models/:id/metadata
-  # Retrieve a trained ML model's metadata
-  def metadata
-    metadata = UserMlModel.where(model_id: params[:id])&.first&.metadata
-    return render_404 unless metadata
-    render json: JSON.parse(metadata)
-  end
-
```

Comment:
Since we don't hit this endpoint, let's remove the ml_models metadata route too, please.

Reply:
Good catch!